docs/package-summary.mdx

```diff
@@ -17,11 +17,11 @@ Depends on @emotion/react
 ## @emotion/css
 
 Contains a framework agnostic version of Emotion. Has different functionality and APIs from @emotion/react and doesn’t work the same way under the hood.
-Depends on @emotion/cache which which can be customised by creating a custom instance at @emotion/css/create-instance
+Depends on @emotion/cache which can be customized by creating a custom instance at @emotion/css/create-instance
```

Comment:
both "customized" and "customised" are correct - the former is used by American English and the latter is British. 

@mitchellhamilton do you have any preference over this? to which flavor Australian is closer?

Reply:
I don't really have a strong preference, American is probably the better one to use to be consistent with what APIs generally use (e.g. `color` in CSS).